tests/tasks/test_task.py

```diff
@@ -16,14 +17,14 @@ async def test_get_sysuptime_returns_uptime(self, snmpsim, snmp_test_port):
             port=snmp_test_port,
         )
         state = ZinoState()
-        task = ReachableTask(device, state)
+        task = ReachableTask(device, state, zino.state.config)
         uptime = await task._get_uptime()
         assert uptime
 
     async def test_get_sysuptime_raises_timeout_error(self):
         device = PollDevice(name="nonexist", address="127.0.0.1", community="invalid", port=666)
         state = ZinoState()
-        task = ReachableTask(device, state)
+        task = ReachableTask(device, state, zino.state.config)
```

Comment:
Why does this task need the global config variable during tests?  Won't an empty (i.e. default values only) `Configuration` object do?